### DISCUSSION TITLE
How to save data to related model

### DISCUSSION THREAD

**Discussion by khairulimran-97** [filamentphp/filament#9996] [Help]
How to add data in Table B when form submit for Table A ?. Only need table_id from A to table B.

**Comment by leandrocfe** [comment]
Create page
https://filamentphp.com/docs/3.x/panels/resources/creating-records#customizing-the-creation-process

Edit page
https://filamentphp.com/docs/3.x/panels/resources/editing-records#customizing-the-saving-process
